### PR TITLE
docs(g11n): add some notes

### LINKED ISSUE #1335: Document support for other languages
We should add some documentation around how folks can use other languages with the Carbon Design System, especially with some of the new type faces coming from IBM Plex 👍

### PR DESCRIPTION
**Pull request by asudoh**
Fixes #1335.

#### Changelog

**New**

- Some notes on G11N.

### CODE REVIEW

File: docs/g11n.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Notes of globalization
+
+* `carbon-components` let you define most of the markups, where most of the translatable contents are in. Application needs to make sure such contents in the markups are translatable as needed.
```
Comment (emyarod) [inline docs/g11n.md]:
```suggestion
* `carbon-components` lets you define most of the markup, which contains most of the translatable content. Ensure application content in the markup is translatable as needed.
```

File: docs/g11n.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+* Also, from the same reason above, application needs to make sure list items in the markup are in the correct order.
```
Comment (emyarod) [inline docs/g11n.md]:
```suggestion
* Also make sure that list items in the application markup are correctly ordered for the same reason.
```

**Review by emyarod — APPROVED** [review]

**Comment by asudoh** [comment]
Thanks @emyarod for reviewing and the corrections!

**Review by tw15egan — APPROVED** [review]
Looks great, thanks for adding in this documentation!

**Pull request merged**